src/components/widget/wrapped-widget.tsx

```diff
@@ -63,12 +67,31 @@ export default function WrappedWidget(props: WrappedWidgetProps) {
     <Box className='widgetHeader'>
       <Flex>
         <Center>
+        <Button
+            variant='link'
+            onClick={onToggle}
+            aria-label={isOpen ? 'downCaret' : 'upCaret'}
+            rightIcon={isOpen ? <ChevronDownIcon/> : <ChevronUpIcon />}
+            size='lg'
+            colorScheme='black'
+          >
+          </Button>
```

Comment:
Can we get this to float left?

Reply:
added it in case, but didn't see that it really had an effect here

Reply:
It would likely need to be outside of the <Center /> component in order for it to work.